### DISCUSSION TITLE
SEO & SMO v3

### DISCUSSION THREAD

**Discussion by slamer59** [saleor/saleor#9960] [Q&A] [answered]
Hello,
I wonder where is SEO & SMO documentation ? [I saw it in v2 ](https://docs.saleor.io/docs/2.11/dashboard/systemwide/seo-smo)and it seems not explictely in v3. It's in the [graphql API](https://docs.saleor.io/docs/3.x/developer/api-reference/inputs/seo-input)

It is a nice feature so, I wonder if you have insights on how to use it. Do you have an exemple in frontend ? 

Thanks :)

**Comment by timuric** [comment, marked as answer]
@slamer59 in a nutshell you can add SEO fields into your product description and pass them to the meta of the page in your template. You can use https://github.com/saleor/react-storefront for reference
